haystack/components/converters/azure.py

```diff
@@ -10,7 +15,7 @@
 logger = logging.getLogger(__name__)
 
 with LazyImport(message="Run 'pip install \"azure-ai-formrecognizer>=3.2.0b2\"'") as azure_import:
```

Comment:
```suggestion
with LazyImport(message="Run 'pip install \"azure-ai-formrecognizer>=3.3.2\"'") as azure_import:
```
Let's update the minimum requirements. It's required to work with the current version of Azure's endpoint.